### ISSUE TITLE
numpy : ep_3, сборка матрицы A

### ISSUE THREAD

**Issue by avasyukov** [ermakov-ad/python_labs#7]
Вот тут:
https://github.com/ermakov-ad/python_labs/blob/ee9d0e8a49670dd2bddb6a8698cd6049aca0dbae/ep_3.py#L9-L12
многовато странного. И в итоге матрица собралась не совсем верно.

Из-за этого ниже по коду потребовалось:
https://github.com/ermakov-ad/python_labs/blob/ee9d0e8a49670dd2bddb6a8698cd6049aca0dbae/ep_3.py#L31

Хотя этот кусочек логики тоже был зашит в вид матрицы. _(Но вообще-то респект, что смогли понять, что происходит, и выкрутиться.)_

И хотя выкрутились в итоге, и картинка ок, но код всё-таки придётся немного поправить.

**Comment by ermakov-ad** [comment]
Да, согласен, матрица А была собрана неверно, не заметил -1 на побочной диагонали. Не смог найти стандартную функцию циклического сдвига матрицы numpy, если такая вообще есть, поэтому исправил текущую сборку
